.github/workflows/cmake_conan.yml

```
@@ -21,7 +21,7 @@ jobs:
         with:
           python-version: "3.10"
       - name: Install Conan
-        run: pip install conan pytest && conan --version
+        run: pip install git+https://github.com/conan-io/conan.git@release/2.0 pytest && conan --version
```

This shouldn't be necessary anymore
```suggestion
        run: pip install conan pytest && conan --version
```